target/sim/sim.mk

```diff
@@ -35,6 +35,9 @@ RUNTIME_DEFINES := +define+HYP_USER_PRELOAD="$(HYP_USER_PRELOAD)"
 RUNTIME_DEFINES += +define+HYP0_PRELOAD_MEM_FILE=\"$(HYP0_PRELOAD_MEM_FILE)\"
 RUNTIME_DEFINES += +define+HYP1_PRELOAD_MEM_FILE=\"$(HYP1_PRELOAD_MEM_FILE)\"
 
+# Runtime-selectable Carfield configuration
+CARFIELD_CONFIG ?= carfield_l2dual_safe_secure_pulp_spatz_periph_can
```

Comment:
Might create conflict with the FPGA flow if defined here

Reply:
If you could have a look [here](https://iis-git.ee.ethz.ch/carfield/carfield-nonfree/-/blob/yt/cfg_top/ci.yml?ref_type=heads#L336-352) and [here](https://iis-git.ee.ethz.ch/carfield/carfield-nonfree/-/blob/yt/cfg_top/ci.yml?ref_type=heads#L451-524) just for double check.

Reply:
This should be resolved.